pkgs/top-level/all-packages.nix

```diff
@@ -15534,6 +15534,8 @@ in
 
   moodle = callPackage ../servers/web-apps/moodle { };
 
+  engelsystem = callPackage ../servers/web-apps/engelsystem { };
```

Comment:
Please keep the list sorted alphabetically

Reply:
I hope I have the right place now

Reply:
Nope, it is divided into several sections and you put it in the wrong section now